modules/querying-bootstrap-node-journal-logs.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/gathering-cluster-data.adoc
+
+[id="querying-bootstrap-node-journal-logs_{context}"]
+= Querying bootstrap node journal logs
+
+If you experience issues with the bootstrap process, you can gather `bootkube.service` `journald` unit logs on a bootstrap node.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the cluster-admin role.
+* Your API service is still functional.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+* Query `bootkube.service` `journald` unit logs from a bootstrap node during {product-title} installation.
+.. Determine the IP address of the bootstrap node.
++
+.. Obtain the `bootkube.service` `journald` unit logs from the bootstrap node. The following example uses bootstrap node IP address '192.168.0.100':
++
+----
+$ curl --insecure --cert <installation_directory>/tls/journal-gatewayd-client.crt --key <installation_directory>/tls/journal-gatewayd-client.key 'https://192.168.0.100:19531/entries?follow&_SYSTEMD_UNIT=bootkube.service'  <1>
```

Comment:
@gpei thank you for your review and for highlighting this. I have now changed this module entirely so that SSH is used to access bootstrap node and container logs. I have also changed the prerequisites in reference to the SSH approach.